scripts/config

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+source /usr/share/yunohost/helpers
+ynh_abort_if_errors
+
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+
+function get__encryption {
+    encryption=$(ynh_app_setting_get --app $app --key encryption)
+    echo "'${encryption}'"
+}
+
+function get__botname {
+    botname=$(ynh_app_setting_get --app $app --key botname)
+    echo "${botname}"
+}
```

Comment:
I'm puzzled as to why these are needed rather than relying on the `bind` mechanism which should read the file content directly ?

Reply:
For the `encryption` one it has to do of the way the manifest return the value after the user selects it. And also the way the config panel expects it (surrounded with `''`). I know it is kinda ugly 😬 

For the `botname` one we have to trigger those commands after it is changed because the new one has to be registered  in Synapse db, so we need getter and setter here 🙂

Reply:
for encryption, is there no cleaner way like changing the manifest or putting some hack in the install script to get a correct value ?
For botname, we could also consider that the yunohost user does not need to change it after installation, or that it is an advanced use-case. If he really wants he can go with the command line, and trigger the registration (this could be explained in the README)

Reply:
For encryption, I could have a look into that, I think I tested in the past without much success but I'll have a further look :)

For botname, maybe you are right, we have to think for a more user friendly config approach, I kinda went all the way to expose everything but it is true that the value of exposing certain config options to the panel it is questionable. Some configs assume that you have certain knowledge to know your way into SSH and other admin stuff so I'm rooting now more to the user friendly approach and expose only non technical stuff :)

Reply:
1. remove `botname` `set__botname` and add instructions in the README 
2. Test and keep `set__botname`